.ibm/images/Dockerfile

```diff
@@ -79,5 +83,9 @@ RUN export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n
     curl -LO "https://github.com/operator-framework/operator-sdk/releases/download/v1.39.1/operator-sdk_${OS}_${ARCH}" && \
     chmod +x operator-sdk_${OS}_${ARCH} && mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
 
+# Ensure Yarn is always activated in interactive shells
+RUN echo "corepack enable" >> /root/.bashrc \
+    && echo "corepack prepare yarn@$(awk -F'[-.]' '/yarnPath:/ {print $(NF-3)"."$(NF-2)"."$(NF-1)}' /root/.yarnrc.yml) --activate" >> /root/.bashrc
+
```

Comment:
can we also echo the yarn version being used here so there's a record of it? would make verifying way easier than having to pull the image :D